### PR TITLE
[1LP][RFR] New tests for Quota: checks quota assigned by automation methods for user and group by provisioning instance over quota limit

### PR DESCRIPTION
**Pull request by ganeshhubale**
- This test case verifies the quota assigned by automation method for user and group
   is working correctly for the cloud providers.
- Steps:
    - Navigate to Automation > Automate > Explorer
    - Add quota automation methods to domain
    - Change 'quota_source_type' to 'user' or 'group'
    - Test quota by provisioning instances over quota limit via UI or SSUI for user and group
    - Check whether quota is exceeded or not

{{ pytest: cfme/tests/cloud/test_quota_tagging.py::test_quota_cloud_via_services -v }}

### CODE REVIEW

File: cfme/tests/cloud/test_quota_tagging.py [hunk 1]
```diff
@@ -215,3 +215,30 @@ def test_cloud_quota_by_lifecycle(request, appliance, provider, setup_provider,
...
+              SSUI for user and group
+           5. Check whether quota is exceeded or not
+    """
+    with appliance.context.use(context):
+        service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+        if context is ViaSSUI:
+            service_catalogs.add_to_shopping_cart()
+        service_catalogs.order()
+    # nav to requests page to check quota validation
+    request_description = 'Provisioning Service [{0}] from [{0}]'.format(catalog_item.name)
```
Comment (digitronik) [inline cfme/tests/cloud/test_quota_tagging.py]: @ganeshhubale please go with `keys`

**Comment by digitronik** [comment]
@ganeshhubale In future please don't add `PR` title in such way. It should be `short` but PR `description` more explanative.

I would like you should read this. It will help you...
https://wiki.openstack.org/wiki/GitCommitMessages#Git_Commit_Good_Practice

File: cfme/tests/cloud/test_quota_tagging.py [hunk 2]
```diff
@@ -215,3 +215,31 @@ def test_cloud_quota_by_lifecycle(request, appliance, provider, setup_provider,
...
+              SSUI for user and group
+           5. Check whether quota is exceeded or not
+    """
+    with appliance.context.use(context):
+        service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+        if context is ViaSSUI:
+            service_catalogs.add_to_shopping_cart()
+        service_catalogs.order()
+    # nav to requests page to check quota validation
+    request_description = "Provisioning Service [{catalog_item__name}] from [{catalog_item__name}]"\
```
Comment (izapolsk) [inline cfme/tests/cloud/test_quota_tagging.py]: REQUIRED: please use () instead of \

File: cfme/tests/cloud/test_quota_tagging.py [hunk 3]
```diff
@@ -215,3 +215,31 @@ def test_cloud_quota_by_lifecycle(request, appliance, provider, setup_provider,
...
+        .format(catalog_item__name=catalog_item.name)
```
Comment (izapolsk) [inline cfme/tests/cloud/test_quota_tagging.py]: Q: why 2 underscores ?

Comment (ganeshhubale) [reply cfme/tests/cloud/test_quota_tagging.py]: It's My bad. 
I changed **catalog_item__name** to **catalog_item_name**.

**Comment by ganeshhubale** [comment]
@izapolsk 
Now tests are not skipping or uncollected on 5.10. It was openstack refresh issue. Now all the tests are passed on 5.10.
Please take a look at it. I have updated with requested changes.
let me know if you have any other suggestions around this PR.

Thank you.

**Pull request merged**